src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1339,6 +1353,25 @@ def _configure_security_policy_storage_params(arg_ctx):
                    'only connections made through Private Links can reach this server.',
                    is_preview=True)
 
+        c.argument('primary_user_assigned_identity_id',
+                   options_list=['--primary-user-assigned-identity-id', '--pid'],
+                   help='The ID of the primary user managed identity.')
+
+        c.argument('key_id',
+                   options_list=['--key-id', '-k'],
+                   help='The key vault URI for encryption.')
+
+        c.argument('user_assigned_identity_id',
+                   options_list=['--user-assigned-identity-id', '-a'],
```

Comment:
In Azure CLI command guideline, -id is GUID as you can see in https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#argument-naming-conventions

Reply:
I posted with more details on the group chat. Please check all feature owners are there on that chat.

Reply:
What about User Assigned Identities. They are ids but not typical ARM ids so we can't split them into name/ids. This parameter is called user_assigned_identity_id everywhere i.e. ARM, Powershell and this is an identity resource id